server/src/main/java/org/elasticsearch/index/query/SearchExecutionContext.java

```diff
@@ -745,22 +743,4 @@ public void setRewriteToNamedQueries() {
     public boolean rewriteToNamedQuery() {
         return rewriteToNamedQueries;
     }
-
-    /**
-     * Returns the minimum lower bound across the time ranges filters against the @timestamp field included in the query
-     */
-    public Long getRangeTimestampFrom() {
-        return rangeTimestampFrom;
-    }
-
-    /**
-     * Records the lower bound of a time range filter against the @timestamp field included in the query. For telemetry purposes.
-     */
-    public void setRangeTimestampFrom(Long rangeTimestampFrom) {
-        if (this.rangeTimestampFrom == null) {
-            this.rangeTimestampFrom = rangeTimestampFrom;
-        } else {
-            this.rangeTimestampFrom = Math.min(rangeTimestampFrom, this.rangeTimestampFrom);
-        }
-    }
```

Comment:
I realized that we need this in QueryRewriteContext, that this extends, to do the same tracking when we parse the time range filter as part of query rewrite. There's cases when the range gets rewritten to a range with no bounds (match all), for which we still want to bucket the request based on its original time range.